lib/internal/shell.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+
+// There is no need to add primordials to this file.
+// `shell.js` is a script only executed when `node run <script>` is called.
+
+const forbiddenCharacters = /[\t\n\r "#$&'()*;<>?\\`|~]/;
+
+/**
+ * Escapes a string to be used as a shell argument.
+ *
+ * Adapted from `promise-spawn` module available under ISC license.
```

Comment:
Do we need to update the license file?

Reply:
I don't think so. My motivation: This is mostly done as a courtesy not as a requirement.